schema/datetime_fmt/test_schema.json

```diff
@@ -0,0 +1,139 @@
+{"$schema": "https://json-schema.org/draft/2020-12/schema",
+ "$id": "https://github.com/unicode/conformance/datetime_test.json'",
+ "title": "ICU Conformance test data description",
+ "description": "This documents the format of date/time test data for execution",
+ "type": "object",
+ "properties": {
+   "additionalProperties": false,
+   "test_type": {
+     "description": "The name of these tests",
+     "type": "string",
+     "enum": ["datetime_fmt"]
+   },
+   "tests": {
+     "description": "list of N tests for date time formatting",
+     "type": "array",
+     "additionalProperties": false,
+     "items": {
+       "properties": {
+         "label": {
+           "description": "A numeric ID, unique for the set of tests",
+           "type": "string"
+         },
+         "locale": {
+           "description": "language tag for formatting the output",
+           "type": "string"
+         },
+         "input": {
+           "description": "date time specification in XYZ format",
+           "type": "string"
+         },
+         "input_millis": {
+           "description": "numeric value for time in milliseconds since January 1, 1970, UTC (the epoch)",
+           "type": "string"
+         },
+         "input_string": {
+           "description": "String in ISO 8601 form, e.g. YYYY-MM-DD hh:mm:ss.sss",
+           "type": "string"
+         },
```

Comment:
Are all three of these fields valid input?

Reply:
Removed input and input_millis fields.